src/frontend/sharedComponents/GPSPill.tsx

```diff
@@ -49,17 +37,13 @@ export const GPSPill: FC<GPSPillProps> = ({
 
   return (
     <TouchableOpacity
-      onPress={onPress}
+      onPress={props.onPress}
       style={styles.container}
-      testID="MAP.gps-pill"
-      accessibilityLabel="Open GPS Modal">
+      testID={props.testID}
+      accessibilityLabel={props.accessibilityLabel}>
       <IconToRender />
 
-      <BodyText
-        variant="smallMeta"
-        style={styles.text}
-        numberOfLines={1}
-        testID="MAP.gps-pill-text">
```

Comment:
removing this test ID has a small effect on one of the e2e tests. see other comment in relevant test file.